ratis-server/src/main/java/org/apache/ratis/server/impl/RaftConfigurationImpl.java

```diff
@@ -288,6 +284,11 @@ public long getLogEntryIndex() {
     return peers.stream().filter(p -> !containsInConf(p.getId())).collect(Collectors.toList());
   }
 
+  /** @return the peers which are not contained in conf. */
+  Collection<RaftPeer> filterListenerNotContainedInConf(List<RaftPeer> listeners) {
+    return listeners.stream().filter(p -> !containsListenerInConf(p.getId())).collect(Collectors.toList());
+  }
+
```

Comment:
Let's add this method later since we need to thing about the new logic.

Reply:
will do